app/components/scheduled_post_indicator_with_datetime/index.tsx

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {withDatabase, withObservables} from '@nozbe/watermelondb/react';
+import {switchMap} from 'rxjs/operators';
+
+import {observeScheduledPostsForTeam} from '@queries/servers/scheduled_post';
+import {observeCurrentTeamId} from '@queries/servers/system';
+
+import ScheduledPostIndicatorWithDatetime from './scheduled_post_indicator_with_datetime';
+
+import type {WithDatabaseArgs} from '@typings/database/database';
+
+type Props = WithDatabaseArgs & {
+    channelId: string;
+}
+
+const enchanced = withObservables(['channelId'], ({database}: Props) => {
```

Comment:
why do you need channelId here if is not being used at all.

Reply:
Missed it. Will remove it. Nice catch 🙌